src/Relude/Foldable/Fold.hs

```diff
@@ -179,9 +179,7 @@ False
 andM :: (Foldable f, Monad m) => f (m Bool) -> m Bool
 andM = foldr go (pure True)
   where
-    go p acc = do
-        q <- p
-        if q then acc else pure False
+    go p acc = ifM p acc (pure False)
```

Comment:
Thanks for simplifying the code!

While we are on this, could you also, please add explicit type signatures to each `go` function? According to our style guide, each definition inside `where` clause should have explicit type signature. But for some unknown reasons functions in this module don't follow this rule :thinking:

Reply:
Done. (I'm guessing that the "unknown reasons" are that it requires `ScopedTypeVariables`, which we probably didn't have enabled back when this was written.)